Dockerfile

```diff
@@ -1,27 +1,26 @@
+# For more information, please refer to https://aka.ms/vscode-docker-python
 FROM python:3.11-slim
 
-ENV DEBIAN_FRONTEND=noninteractive
-ENV APPLICATION_ROOT_PATH="/opt/ctb/"
-ENV VAR_PATH="/var/ctb/"
-
-RUN apt update && \
-    apt upgrade -y && \
-    apt install -y sqlite3 libsasl2-dev python3-dev libldap2-dev libssl-dev
-
-RUN mkdir -p $APPLICATION_ROOT_PATH
-RUN mkdir -p $VAR_PATH
-WORKDIR $APPLICATION_ROOT_PATH
+EXPOSE 8080
 
-COPY src/server/requirements.txt /tmp/requirements.txt
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE=1
+# Turns off buffering for easier container logging
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONOPTIMIZE=TRUE
 
-RUN pip3 install --upgrade pip
-RUN pip3 install -r /tmp/requirements.txt
+# Install pip requirements
+COPY src/server/requirements.txt .
+RUN python -m pip install --upgrade pip
+RUN python -m pip install -r requirements.txt
 
-COPY src $APPLICATION_ROOT_PATH/src
-COPY res $APPLICATION_ROOT_PATH/res
+WORKDIR /app
+COPY . /app
```

Comment:
We don`t need to copy everything to the image - please either update dockerignore or change it here

Reply:
Done